MANIFEST.in

```diff
@@ -0,0 +1,3 @@
+include README.rst
+include LICENSE
+recursive-include allauth_2fa/templates *
```

Comment:
Ah, I see these aren't packaged yeah. We completely override them in our package, so I didn't notice. Good catch.